prep-html-links/contact-us/contact-us.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Contact Us</title>
+</head>
+<body>
+  <h1>Contact Us</h1>
+  <!-- <div>Click here to go to the home page</div> -->
+  <a href="C:\Users\chris\repos\prep-90921-code-solutions\prep-html-links\index.html">Click here to go to the home page</a>
```

Comment:
This is what is known as an absolute file path and will only work on your computer. In our code we want to avoid using absolute file paths because they are very system specific and we want our code to work on any system. Here we want to use a relative path. A relative is the direct path from one file to another file. So here your `href` should be set to `../index.html`